backend/fms_core/template_importer/row_handlers/library_qc/library_qc.py

```diff
@@ -15,43 +15,69 @@ def __init__(self):
         super().__init__()
 
     def process_row_inner(self, sample_container, measures, process, process_measurement, process_measurement_properties):
-         # Get the library sample that was checked
-        source_sample_obj, self.errors['container'], self.warnings['container'] = \
-            get_sample_from_container(barcode=sample_container['container_barcode'], coordinates=sample_container['container_coord'])
 
+        barcode = sample_container['container_barcode']
+        coordinates = sample_container['container_coord']
+
+        if barcode is None:
```

Comment:
I had several exceptions caused by None values (eg. comparing a volume to None) and that ruins the validation and just displays an unhelpful error message in the freezeman page, so I decided to be more aggressize and do my None checks once, rather than having to have None checks all over the place. These checks are to avoid unexpected exceptions blowing up the validation.

Reply:
Can I add a check in the service to explicitly check for a None value for the barcode?

Reply:
Yeah this could improve the actual error message returned and prevent a useless call to get().